docs/01-membership.md

```diff
@@ -14,6 +14,13 @@ Individuals from active working groups produce the membership by opting into Pro
 - 11 Working Groups, 36 contributors
 - Venue: breakout calls
 - Artifacts: Research & POCs
+- Research Constraints
+     - Generally agreed to be a significant and worthwhile direction for the protocol
```

Comment:
I find this constraint to be too broad to effectively constrain. "Generally", "Significant" and "Worthwhile" are all words that lack precision, and thus forfeit enforcement of Ethereum values.  Protocol Guild subsidized research should be required to be value aligned with Ethereum.

For example, in this current example, I agree that the zkEVM working group is "significant and worthwhile", but the centralization threat of provers remains an ignored design space. If an area of research is willing to sacrifice decentralization in favor of scaling, that is a SIGNIFICANT departure from Ethereum values, which would sail right through these constraints.

We need to try harder to protect what we value.

Reply:
I also don't think we should be changing category constraints while at the same time trying to add a working group. Feels like we're changing the rules to support the proposal instead of changing the rules to support the consensus view of the guild.

Reply:
appreciate the feedback, your consistent disapproval on grounds of "insufficient decentralization" is noted 🫡

one counterpoint to consider - the original "Casper the Friendly Finality Gadget" (FFG) design required 1500 ETH requirements and would have likely been excluded from funding had 1. PG existed 2. used your suggested requirements at the time. this work was important to inform the 32 ETH design of the beacon chain the community settled on.

Reply:
Great example! In that case I still would have excluded that research from Protocol Guild, till they got the design to a more decentralized one. Once that was proved, I would admit the working group, and backdate their working weights to the time of group inception.

Reply:
Also eth was <$10 when the original idea of 1500 requirement was being proposed, so not sure you should view it in the context of today's price.

Reply:
not sure that's correct?

- price = $300 when the [original paper](https://arxiv.org/abs/1710.09437) released (Oct 2017)
- price = $750 when the [explainer post](https://vitalik.eth.limo/general/2017/12/31/pos_faq.html) was released (dec 2017)
- price peaked at $1450 in jan 2018
- price = $500 when the the [first mention](https://ethresear.ch/t/safe-notary-pool-size/1728/3) of "32 ETH" on ethresearch (April 2018)

under the period of FFG consideration (using my rough cutoffs), the cost of a single validator would have ranged from $450,000 to $2,175,000

it's easy to look back with hindsight and say "this was unacceptable", when this is was the tradeoff space core devs found themselves designing in. We will surely do the same in 5 years when the limit is lower than 32 ETH 🤞